lambdas/check-existing-identity/src/test/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandlerTest.java

```diff
@@ -751,9 +749,57 @@ void shouldReturn500IfCredentialParseExceptionFromCheckBirthDateCorrelationInCre
 
     @Test
     void shouldReturnCiJourneyResponseIfPresent() throws Exception {
+        var testCiCode = "TEST01";
+        var testJourneyResponse = "/journey/test-response";
+        var testContraIndicators =
+                ContraIndicators.builder()
+                        .contraIndicatorsMap(
+                                Map.of(
+                                        testCiCode,
+                                        ContraIndicator.builder().code(testCiCode).build()))
+                        .build();
+        var testCimitConfig = Map.of(testCiCode, testJourneyResponse);
+
+        when(ipvSessionService.getIpvSession(TEST_SESSION_ID)).thenReturn(ipvSessionItem);
+        when(ciMitService.getContraIndicatorsVC(
+                        TEST_USER_ID, TEST_JOURNEY_ID, TEST_CLIENT_SOURCE_IP))
+                .thenReturn(testContraIndicators);
+        when(userIdentityService.breachingCiThreshold(testContraIndicators)).thenReturn(true);
+        when(configService.getCimitConfig()).thenReturn(testCimitConfig);
+
+        when(clientOAuthSessionDetailsService.getClientOAuthSession(any()))
+                .thenReturn(clientOAuthSessionItem);
+
+        JourneyResponse response =
+                toResponseClass(
+                        checkExistingIdentityHandler.handleRequest(event, context),
+                        JourneyResponse.class);
+
+        assertEquals(testJourneyResponse, response.getJourney());
+    }
+
+    @Test
+    void shouldNotReturnCiJourneyResponseForMitigatedCi() throws Exception {
+        var testCiCode = "TEST01";
+        var testJourneyResponse = "/journey/test-response";
+        var testContraIndicators =
+                ContraIndicators.builder()
+                        .contraIndicatorsMap(
+                                Map.of(
+                                        testCiCode,
+                                        ContraIndicator.builder()
+                                                .code(testCiCode)
+                                                .mitigation(List.of(Mitigation.builder().build()))
+                                                .build()))
+                        .build();
+        var testCimitConfig = Map.of(testCiCode, testJourneyResponse);
```

Comment:
Nit-picky/ not important but could these variables be hoisted to the class if they can be used for both of these tests. Is more similar to what's been done for the other tests in this file

Reply:
They could - I don't know how useful it is. Semantically I think they are local variables (they can be different in each test and it'd still be valid), but hoisting them saves redeclaring them. I also quite like self-contained tests...